Views/Shared/_PlayerGraphPartialScripts.cshtml

```diff
@@ -23,6 +25,10 @@
             },
             function(data){
                 if (dataKind2 < 254) {
+                // For dual graphs I need to create 4 combinations of y-axis settings
```

Comment:
Yeah, it could be done simpler. What I would do is: I would create a separate variable outside of the request code (next to where the other var's are declared in this JS snippet) to hold the axis configuration and move the conditions there.